### PR TITLE
Feature/pt br translation

### PR DESCRIPTION
**Pull request by fcpimenta**
## Type of Change

- Add Portuguese Brazilian translation

## What issue does this relate to?

- N/A

### CODE REVIEW

**Comment by MattIPv4** [comment]
👋 Tyvm for the PR, @fcpimenta!

Could you rebase this against lastest master when you have a second -- we just merged a few new i18n strings and a tool to help check i18n packs :)

**Comment by fcpimenta** [comment]
> 👋 Tyvm for the PR, @fcpimenta!
> 
> Could you rebase this against lastest master when you have a second? -- we just merged a few new i18n strings and a tool to help check i18n packs :)

Hi Matt! I've already rebased my fork. Could you please check if all things are ok?

**Comment by MattIPv4** [comment]
It looks like this branch specifically still needs to be rebased against your fork's master branch

**Comment by fcpimenta** [comment]
Branch rebased.

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Ty for the rebase :)

Could you export the pack in `src/nginxconfig/i18n/index.js`, please?

This should let the CI then begin to validate all the pack translation keys are present.

**Comment by fcpimenta** [comment]
I think the issues were corrected.

**Review by botreview — APPROVED** [review]
LGTM

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Tyvm!! Just a few small things :)

File: src/nginxconfig/i18n/pt-br/common.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	letsEncrypt: 'Let\'s Encrypt',
+	python: 'Python',
+	docker: 'Docker',
+	dockerCompose: 'Docker Compose',
+	wordPress: 'WordPress',
+	drupal: 'Drupal',
+	magento: 'Magento',
+	joomla: 'Joomla',
+	django: 'Django',
+	logging: 'Logs',
```
[the remark this answers is not in the document]
Comment (fcpimenta) [reply src/nginxconfig/i18n/pt-br/common.js]: I've verified all indentation. Here there is no error.

[the remark this answers is not in the document]
Comment (MattIPv4) [reply src/nginxconfig/i18n/pt-br/common.js]: Weird, looks like there were some tabs instead of spaces, just committed a quick fix

**Review by MattIPv4 — APPROVED** [review]
Lgtm, tyvm!

**Comment by MattIPv4** [comment]
Hey, @fcpimenta - Thanks a ton for this sweet PR! 😄

Would you please shoot me an email when you get a chance?
mcowley at digitalocean dot com 🎉

**Pull request merged**